### PR TITLE
feat(metadata-service): add consent form (DSP-1704)

### PR DESCRIPTION
**Pull request by mpro7**
resolves DSP-1704

Accepting both changes saves cookie which expires after 90 days. Accepting "necessary cookies" prevents GA script to be fired - no data are collected. Choosing "all" option enables GA. Modal is shown only if corresponding cookie doesn't exist.

### CODE REVIEW

**Review by subotic — APPROVED** [review]

File: services/metadata/frontend/CookiesModal.svelte [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+      <div class="modal-content-wrapper">
+        <div class="modal-content">
+          <div class="modal-header">
+            <h3 class="modal-title">Cookies settings</h3>
+          </div>
+          <div class="modal-body">
+            We use cookies to personalize content and analyze access to our websites. Please choose whether you 
+            accept only cookies that are necessary for the functioning of the website or whether you accept also
+            analytical cookies. For more information, please refer to our
+            <a href="https://dasch.swiss/privacy-policy/" target=_blank>privacy policy</a>.
```
Comment (subotic) [inline services/metadata/frontend/CookiesModal.svelte:30]: Could you please also add links to the EULA: dasch.swiss/eula/ and Cookie Policy: dasch.swiss/cookie-policy?

Comment (mpro7) [reply services/metadata/frontend/CookiesModal.svelte:32]: Done in c1d7e78.

Comment (mpro7) [reply services/metadata/frontend/CookiesModal.svelte:32]: Actually should we call it `cookie` or `cookies policy`? To make it consistent I either change modal header where it's `cookies settings` now or link and description need to be changed.

Comment (subotic) [reply services/metadata/frontend/CookiesModal.svelte:32]: sorry, I wasn't very clear. We will probably have all three: EULA, privacy policy, and cookies policy. @danielasubotic is going to check what exactly is needed next week. Maybe we can merge privacy policy and cookies policy into one document.

**Pull request merged**